common/lib/xmodule/xmodule/js/src/video/08_video_speed_control.js

```diff
@@ -38,7 +38,7 @@ define(
                     'title="',
                         gettext('Adjust video speed'),
                     '" aria-describedby="speed-instructions">',
-                    '<span class="icon-fallback-img">',
+                    '<span>',
                         '<span class="icon fa fa-caret-right" aria-hidden="true"></span>',
                         '<span class="sr control-text">',
```

Comment:
Since this button has a `title` we can simplify this and remove the `sr` text within it.